src/modules/cmdpal/Microsoft.CmdPal.UI/Settings/ExtensionPage.xaml

```diff
@@ -41,8 +41,8 @@
             <RowDefinition Height="Auto" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
-        <ScrollViewer Grid.Row="1" Padding="0,0,8,0">
-            <Grid Padding="8,-8,8,8">
+        <ScrollViewer Grid.Row="1">
+            <Grid Padding="16,-8,16,16">
```

Comment:
```suggestion
            <Grid Padding="16">
```

This way it's consistent and the top is not getting cut off (happens on `main` too btw)